main.go

```diff
@@ -86,7 +86,10 @@ func main() {
 	}
 
 	// Attempt to find the license key for telemetry sending
-	licenseKey, err := credentials.GetNewRelicLicenseKey(ctx, conf)
+	var timeout = 5 * time.Second
```

Comment:
I think 5 seconds is a really long time. Maybe 1 second is appropriate here?

Reply:
Modified to 1 second in the revision. Thanks!